apps/web/src/sanity/job-ad/queries.ts

```diff
@@ -3,10 +3,11 @@ import groq from "groq";
 export const jobAdsQuery = groq`
 *[_type == "job"
   && !(_id in path('drafts.**'))]
-  | order(_createdAt desc) {
+  | order(deadline desc) | order(weight desc) {
```

Comment:
Tror vi vil ha det andre veien
```suggestion
  | order(weight desc, deadline desc) {
```

Reply:
Jeg er litt usikker på denne da, men du kan prøve så ser du.